### PR TITLE
Prepackage Apps plugin and enable Apps feature flag by default

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

This PR adds the Apps plugin to the list of pre-packaged plugins, as well as defaulting the `AppsEnabled` feature flag to `true`.

At the moment, the PR is using `v0.7.0` as the plugin version, in order to have the CI builds pass. The PR will be changed to use `v1.0.0` once the version has been released.

#### Release Note

```release-note
The server release now pre-packages the Apps Framework plugin, and enable Apps feature flag by default.
```

### CODE REVIEW

**Comment by amyblais** [comment]
@mickmister @DHaussermann Is this for v6.6?

**Comment by amyblais** [comment]
@mickmister I added you to the v6.6 channel where I posted the Code Freeze earlier (April 7th).

File: Makefile [hunk 1]
```diff
@@ -130,6 +130,7 @@ PLUGIN_PACKAGES += mattermost-plugin-nps-v1.1.0
 PLUGIN_PACKAGES += mattermost-plugin-welcomebot-v1.2.0
 PLUGIN_PACKAGES += mattermost-plugin-zoom-v1.5.0
 PLUGIN_PACKAGES += focalboard-v0.15.0
+PLUGIN_PACKAGES += mattermost-plugin-apps-v0.7.0
```
Comment (catalintomai) [inline Makefile:133]: 0.7.0 -> 1.0.0 ?

Comment (mickmister) [reply Makefile]: @catalintomai Fixed :+1:

**Comment by catalintomai** [comment]
> Failed running enterprise tests. @mattermost/core-build-engineers have been notified. Error: `500: An internal server error occurred.`

@mickmister - maybe because of this: https://status.circleci.com/ ?

**Review by hanzei — APPROVED** [review]
:tada:

**Review by catalintomai — APPROVED** [review]

**Review by levb — APPROVED** [review]

**Review by hanzei — APPROVED** [review]

**Comment by hanzei** [comment]
@DHaussermann Ready for your review

**Comment by DHaussermann** [comment]
Tested and passed
- Plugin is installed and enabled when server is instantiated
- Version correctly bumped to `v1.0.1`
- Apps Plugin is functional 
- Tested installing enabling / disabling an app with the Apps framework
- Tested using an app using the Apps framework
- No issue disabling the plugin
- No issues uninstalling the plugin
- No issues re-installing the plugin **without remote marketplace** disabled  
- As a precaution re-tested that there is no need for setting a feature flag to run the plugin

LGTM!

**Pull request merged**